config/manifests/bases/ocs-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: operators.coreos.com/v1alpha1
```

Comment:
Why we got the new CSV file here?

Reply:
my branch is older, so before fusion addition, it was like this. So, I removed the file now. Thanks